test/pkg/gitea/test.go

```diff
@@ -333,12 +334,28 @@ func CheckIfPipelineRunsCancelled(t *testing.T, topts *TestOpts) {
 
 func GetStandardParams(t *testing.T, topts *TestOpts, eventType string) (repoURL, sourceURL, sourceBranch, targetBranch string) {
 	t.Helper()
-	prs, err := topts.ParamsRun.Clients.Tekton.TektonV1().PipelineRuns(topts.TargetNS).List(context.Background(), metav1.ListOptions{
-		LabelSelector: keys.EventType + "=" + eventType,
-	})
-	assert.NilError(t, err)
-	assert.Equal(t, len(prs.Items), 1, "should have only one "+eventType+" pipelinerun")
+	var err error
+	prs := &v1.PipelineRunList{}
+	for i := 0; i < 21; i++ {
```

Comment:
ah okok this is the reason

but why 21 ? i mean its just some time right ?

Reply:
why 21? because i guess it's the legal age of drinking in the United States and that bug makes paac drunk 🙃